docs/proposals/external_providers_grpc.md

```diff
@@ -54,3 +54,22 @@ https://github.com/gardener/machine-controller-manager/tree/grpc-driver
 
 Refer GRPC based implementation for AWS client:
 https://github.com/ggaurav10/aws-driver-grpc
+
+## Alternatives:
+### Cloud Provider as GRPC server:
```

Comment:
Cloud Provider would not be the right terminology. Cloud Provider means something that is already provided.

This should be something else. Driver/CloudProviderManager/CloudProviderExtension...